spec/GraphQLOverHTTP.md

```diff
@@ -122,17 +122,12 @@ Specification compliant HTTP client unless the context indicates otherwise.
 
 # URL
 
-A server MUST enable GraphQL requests to one of more GraphQL schemas.
+A server MUST enable GraphQL requests to one or more GraphQL schemas.
 
 Each GraphQL schema a server provides MUST be served via one or more URLs.
 
-A client MUST be able to send all their GraphQL query and mutation requests for
-a single GraphQL schema made available by a server to a single URL endpoint on
-that server. A server MUST NOT require the client to use different URLs for
-different GraphQL query and mutation requests to the same GraphQL schema.
-
-Note: This means that a client must be able to issue all GraphQL query and
-mutation operations it needs via a single endpoint.
+A server MUST NOT require the client to use different URLs for different GraphQL
+query and mutation requests to the same GraphQL schema.
```

Comment:
By mentioning these specifically, it might create confusion as to why subscriptions are not mentioned.

```suggestion
A server MUST NOT require the client to use different URLs for different GraphQL
requests to the same GraphQL schema.
```